src/LangStrings.inc.h

```diff
@@ -236,6 +236,12 @@ DECLARE_STRING(PLANET_WITH_SOME_ICE)
 DECLARE_STRING(MOON_WITH_SOME_ICE)
 DECLARE_STRING(ROCKY_PLANET_CONTAINING_COME_LIQUIDS)
 DECLARE_STRING(ROCKY_MOON_CONTAINING_SOME_LIQUIDS)
+DECLARE_STRING(DWARF_PLANET_ICE)
+DECLARE_STRING(DWARF_PLANET_WATER)
+DECLARE_STRING(DWARF_PLANET_CONTAINING_LIQUID_WATER)
+DECLARE_STRING(DWARF_PLANET_WITH_SOME_ICE)
+DECLARE_STRING(DWARF_PLANET_TERRESTRIAL_CONTAINING_SOME_LIQUIDS)
+DECLARE_STRING(DWARF_PLANET_TERRESTRIAL)
```

Comment:
I don't see JSON language strings defined for `DWARF_PLANET_WATER`, `DWARF_PLANET_CONTAINING_LIQUID_WATER`, `DWARF_PLANET_WITH_SOME_ICE`. Left-overs, or partially-implemented?

Reply:
I think those got lost when I rebased the branch ahead of the push. I'll look into it.

Reply:
`DWARF_PLANET_WATER` -> Removed
`DWARF_PLANET_CONTAINING_LIQUID_WATER` -> Implemented
`DWARF_PLANET_WITH_SOME_ICE` -> On hold for now. I think there are enough new strings.

Reply:
I also sorted the strings so they are more easy to compare.